_assets/js/touchpoints-page-helpful.js

```diff
@@ -0,0 +1,20 @@
+export default function tryToSetPageAnswer() {
+    if (!document.querySelector('#touchpoints-yes-no-form')) return;
+
+    const pageAnswer = document.querySelector('#touchpoints-yes-no-form #answer_02');
+    if (!pageAnswer) {
+      setTimeout(tryToSetPageAnswer, 500);
+      return;
+    }
+
+    const answer1 = document.querySelector('#touchpoints-yes-no-form #answer_01');
```

Comment:
supernit - wdyt about naming this "yesNoButtons" or something instead of answer1 to give a bit more context to future readers